code/modules/mining/mine_turfs.dm

```diff
@@ -347,6 +351,42 @@ var/list/mineral_can_smooth_with = list(
 
 /turf/simulated/mineral/proc/GetDrilled(var/artifact_fail = 0)
 	//var/destroyed = 0 //used for breaking strange rocks
+
+	var/turf/T = get_turf(src)
+
+	switch(trap_type)
+		if(SPACETRAP_SHANK)
```

Comment:
What is the purpose of having this switch between A and B if B is just A but with some additional stuff added?